platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/SwirldStateManagerImpl.java

```diff
@@ -80,12 +80,12 @@ public class SwirldStateManagerImpl implements SwirldStateManager {
     /**
      * Handles system transactions pre-consensus
      */
-    private final PreConsensusSystemTransactionManager preConsensusSystemTransactionManager;
+    private final PreconsensusSystemTransactionManager preConsensusSystemTransactionManager;
```

Comment:
while you're here, you might as well change the member variable to `preconsensusSystemTransactionManager`

Reply:
fixed